plugins/gpio/GPIOPort.cpp

```diff
@@ -19,6 +19,7 @@
  */
 
 #include "plugins/gpio/GPIOPort.h"
+#include "ola/StringUtils.h"
```

Comment:
This needs to be just above #include "plugins/gpio/GPIODriver.h" rather than up here please.